src/models/measurement.ts

```diff
@@ -49,6 +53,8 @@ interface Measurement {
   orderOfMagnitude: number;
   significand: number;
   performance: number;
+  anomalousDetector?: 0 | 1 | null;
+  anomalousUser ?: 0 | 1 | null;
```

Comment:
These keys are non-optional, as they are either provided by the backend or by the default in Joi. So the `?` should go in both lines.